src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const AMOUNT_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const baseAmount = days * AMOUNT_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return baseAmount - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return baseAmount - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
There is a logical issue in the discount application. The current implementation applies the long-term discount first, which is correct. However, if the number of days is 7 or more, it will not check for the short-term discount, which is fine since the long-term discount is larger. But if the number of days is exactly 3 to 6, it will apply the short-term discount correctly. Ensure that the logic aligns with the intended discount policy. If the logic is intended to apply only one discount, this implementation is correct.